examples/wing-fixture/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "wing-fixture",
+  "version": "0.0.0",
+  "description": "A fake library just for testing stuff out",
+  "author": {
+    "name": "Wing Cloud",
+    "email": "ping@monada.co",
+    "url": "https://wing.cloud",
+    "organization": true
+  },
+  "files": [
+    "**/*.js",
+    "**/*.w",
+    "!/target/**"
+  ],
+  "license": "MIT",
+  "engines": {
+    "node": ">=v18.0.0"
```

Comment:
We should embed the wing version here

Reply:
I've updated `wing pack` so that a `wing` field is added in the the staging directory / in the final tarball